### PR TITLE
bugfixes for login process

### PR DESCRIPTION
**Pull request by kelvinndmo**
- Bugfixes and improvements for login process

### CODE REVIEW

File: routes/login.py [hunk 1]
```diff
@@ -10,26 +10,32 @@
 router = APIRouter()
 
 
-@router.post("", operation_id="login_user", status_code=status.HTTP_200_OK)
+@router.post(
```
Comment (linerocks) [inline routes/login.py:13]: @wilsonangara adjust this endpoint so that the hospital can use the same endpoint

Comment (wilsonangara) [reply routes/login.py:13]: got it, I will take a look at what's required for all of our users to be able to login using 1 endpoint

File: routes/user.py [hunk 2]
```diff
@@ -79,10 +89,11 @@ async def update_user(user_id: str, payload: UpdateUserPayload):
                 status_code=status.HTTP_400_BAD_REQUEST,
                 detail="Phone number already exists",
             )
+        code = random.randint(100000, 999999)
```
Comment (wilsonangara) [inline routes/user.py:92]: maybe we can have a better naming for this?

**Review by wilsonangara — COMMENTED** [review]
Other than that, LGTM

**Pull request merged**